rotkehlchen/api/rest.py

```diff
@@ -2695,20 +2694,28 @@ def delete_blockchain_transaction_data(
             with self.rotkehlchen.data.db.user_write() as write_cursor:
                 write_cursor.execute(querystr, bindings)
 
+        if not chain or chain == SupportedBlockchain.BITCOIN:
+            with self.rotkehlchen.data.db.user_write() as write_cursor:
+                self.rotkehlchen.data.db.delete_dynamic_caches(
```

Comment:
so what this would do is to reset the cache and transactions will be requeried for the range? This should be documented in the api docs too